### DISCUSSION TITLE
Add a Useful Servers Channel In Discord

### DISCUSSION THREAD

**Discussion by ghost** [EddieHubCommunity/support#1679] [Ideas] [answered]
Add a #useful-servers channels in the discord server. We can add similar communities like Bash Women Community. This would be really great for discord newbies and people who want to join similar servers 👍🏻 🙂 

Example of this: https://discord.gg/bRCvFy9

**Comment by Radon333** [comment]
https://discord.gg/BKXSVp87Ws Web.dev

**Comment by eddiejaoude** [comment, marked as answer]
I think cross community collabs are great. However, the danger with this is it get abused and becomes spammy, which has happened already which is why we removed twitter from the allowed links. 

We do have a `#content-share` channel already for *regulars* to share their links, plus on our EddieHun website there are links to our community partners.
